### PR TITLE
only submit previous files in output only problem when the task mode is SCORE_MODE_MAX_TOKENED_LAST

### LINKED ISSUE #6: Avoid automatically submitting files for output only
For output-only, CMS was designed to automatically submit the last submitted files for output files that is not submitted on the current submission. This feature should not be useful with the IOI 2017 scoring, and might actually be confusing instead. We might want to remove this feature.

### PR DESCRIPTION
**Pull request by aguss787**
Resolves #6, but retain compatibility with other scoring method

### CODE REVIEW

File: cmstestsuite/unit_tests/grading/tasktypes/OutputOnlyTest.py [hunk 1]
```diff
@@ -29,8 +29,8 @@
     OUTCOME, TEXT, TaskTypeTestMixin
 
 
-FILE_001 = File(digest="digest of 001", filename="output_001.txt")
-FILE_023 = File(digest="digest of 023", filename="output_023.txt")
+FILE_001 = File(digest="digest of 001", filename="001.out")
```
Comment (fushar) [inline cmstestsuite/unit_tests/grading/tasktypes/OutputOnlyTest.py:32]: Sorry I really don't understand, what is the purpose of these changes?

Comment (aguss787) [reply cmstestsuite/unit_tests/grading/tasktypes/OutputOnlyTest.py:32]: Because in the output only specifications, the output filename is "%s.out". however, the testcase is output_%s.txt.

This difference cause the test to fail.

**Review by fushar — APPROVED** [review]

File: cmstestsuite/unit_tests/grading/tasktypes/OutputOnlyTest.py (hunk 1 above)
Comment (jonathanirvings) [reply cmstestsuite/unit_tests/grading/tasktypes/OutputOnlyTest.py]: For documentation purposes: these changes are reverted in this PR since they are beyond the scope of this PR.
